css/mystyle.css

```diff
@@ -223,7 +249,16 @@ color: #838994;
     .social{
     	padding: 3em 0;
     }
+    .assign .illustration li img {
+		max-width: 100%;
+		height: 100%;
+	}
 }
-.clear{
-	clear: both;
-}
\ No newline at end of file
+
+@media only screen and (max-width: 417px) {
+	.top-navigation {
```

Comment:
This makes the top area a little bit taller to allow for the navigation links stacking on top of each other.